nri/index.html

```diff
@@ -1 +1,207 @@
 <!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="utf-8">
+        <title>NRI</title>
+        <meta name="discription" content="NRIの公式ホームページです">
+
+    <!--CSS-->
+        <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Philosopher" rel="stylesheet">
+        <link rel="icon" type="image/png" href="images/NRI.png">
+    </head>
+
+    <body>
+          <header class="page-header">
+              <h1><a href="#"><img class="logo" src="images/NRI.png" alt="NRIロゴ"></a></h1>
+              <nav>
+                <ul class="main-nav">
+                    <li><a href="#">ナレッジ・インサイト</a></li>
+                    <li><a href="#">ソリューション・サービス</a></li>
+                    <li><a href="#">サステナビリティ</a></li>
+                    <li><a href="#">ニュース</a></li>
+                    <li><a href="#">企業情報</a></li>
+                    <li><a href="#">株主・投資家情報（IR）</a></li>
+                    <li><a href="#">採用情報</a></li>
+                </ul>
+                <img src="images/search.png" alt="検索">
+                <img src="images/maru.png" alt="言語">
+                <img src="images/san.png" alt="メニュー">
+              </nav>
+          </header>
+          <ul class="log">
+            <li><a href="#">NRI トップ</a></li>
+            <li><a href="#">企業情報</a></li>
+            <li>NRIの強み：NRIのデジタルビジネス戦略</li>
+          </ul>
+          <div class="wrapper">
+            <h3 class="white-1">NRIの強み</h3>
+            <h1 class="white-2">NRIのデジタルビジネス戦略</h1>
+          </div><!--/.wrapper-->
+          <article>
+            <header class="post-info">
+              <h2 class="post-title">NRIのデジタルビジネス戦略</h2>
+            </header>
+            <p>
+              NRI では、企業の DX に向けた取り組みについて、既存ビジネスの業務プロセス変革に寄与する DX を
+              「DX1.0」、デジタルで新しいビジネスモデルそのものを生み出す DX を「DX2.0」と定義しています。
+            </p>
+            <p>
+              DX を実現するためには、業務プロセス変革やビジネスモデル創造、そして、それらを実現する IT を同時並行
+              で、共に議論しながら、かつ、スピーディーに実装できる戦略パートナーが必要です。NRI は、コンサルティン
+              グと IT ソリューションの人材と機能を併せ持ち、お客さまと併走し、お客さまと仮説検証を繰り返しながらビ
+              ジネスを創出する「コンソリューション」の提供を強みとして、DX の実現を推進します。
+            </p>
+            <img src="images/story.png" alt="流れ">
+            <img src="images/pdf.png" alt="ボタン">
+          </article>
+          <div class="link-wrapper">
+          </div><!--/.link-wrapper-->
+          <div>
+            <p><a href="#">ビジネスモデルの特徴</a></p>
+            <p><a href="#">人材が支えるNRI</a></p>
+          </div>
+          <div class="wrapper grid">
+            <p>NRIの強み</p>
+            <div class="item"><a href="#">NRIのあゆみと持続的な成長</a></div>
+            <div class="item"><a href="#">問題発見から問題解決まで</a></div>
+            <div class="item"><a href="#">強固な顧客基盤</a></div>
+            <div class="item"><a href="#">ビジネスモデルの特徴</a></div>
+            <div class="item"><a href="#">人材が支えるNRI</a></div>
+            <div class="item"><a href="#">業界トップクラスの集客力</a></div>
```

Comment:
「NRIの強み」という情報に対して、強みの一つはこれ、も一つはこれ、と説明しているブロックです。

このような場合は、divを使うよりも箇条書き（ul > li）を使うのがよりよいです。
※本ちゃんのサイトはdivですね…w

```suggestion
            <p>NRIの強み</p>
            <ul>
              <li class="item"><a href="#">NRIのあゆみと持続的な成長</a></li>
              <li class="item"><a href="#">問題発見から問題解決まで</a></li>
              ..以下略
            </ul>              
```